src/corehost/cli/dll/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ else()
     add_compile_options(-fPIC)
 endif()
 
+add_compile_options(-fvisibility=hidden)
```

Comment:
Instead of duplicating this in all 5 CMakeLists, can you instead define it in a common place?  Ex. https://github.com/dotnet/core-setup/blob/master/src/settings.cmake.

Reply:
That shared file appears to fairly generic so I kept the setting in each project, close to the fPIC setting which is related